be/src/common/config.h

```diff
@@ -257,6 +258,8 @@ CONF_Int32(storage_page_cache_shard_size, "16");
 CONF_Int32(index_page_cache_percentage, "10");
 // whether to disable page cache feature in storage
 CONF_Bool(disable_storage_page_cache, "false");
+// whether to disable row cache feature in storage
+CONF_Bool(disable_storage_row_cache, "false");
```

Comment:
If row cache do not used in most cases, please disable it by default.

Reply:
row cache is used for row store, row store  is disabled by default, so for most cases this flag is unused